backend.py

```diff
@@ -182,9 +185,16 @@ def lyrics_genius(artist, song):
     return(lyrics, url, timed)
 
 def getwindowtitle():
-    if os.name == "nt":
+    if sys.platform== "win32":

```

Comment:
Sorry about the missing space here. I'm leaving it for others to fix to avoid cluttering here with my tiny commits.